# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxDWriteFontList.h
# Commit: a01eaeed432a
# Full Hash: a01eaeed432ab996a3f5275229be89814e92b95a
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1708768
# File Overlap Count: 2
# Description:
#   Bug 1708768 - Preliminary cleanup of gfxPlatformFontList creation/initialization code; no functional change. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D114863
# ==============================================================================

diff -r 39c32384d7a9 -r a01eaeed432a gfx/thebes/gfxDWriteFontList.h
--- a/gfx/thebes/gfxDWriteFontList.h	Tue May 18 10:47:43 2021 +0000
+++ b/gfx/thebes/gfxDWriteFontList.h	Tue May 18 11:10:42 2021 +0000
@@ -369,7 +369,8 @@
   gfxDWriteFontList();
 
   static gfxDWriteFontList* PlatformFontList() {
-    return static_cast<gfxDWriteFontList*>(sPlatformFontList);
+    return static_cast<gfxDWriteFontList*>(
+        gfxPlatformFontList::PlatformFontList());
   }
 
   // initialize font lists